introduction.md

```diff
@@ -0,0 +1 @@
+One time I hired a monkey to take notes for me in class.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.